core/src/test/java/org/testcontainers/junit/DockerComposeV2Test.java

```diff
@@ -0,0 +1,46 @@
+package org.testcontainers.junit;
+
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.utility.TestEnvironment;
+import org.testcontainers.utility.TestcontainersConfiguration;
+import redis.clients.jedis.Jedis;
+
+import java.io.File;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
+
+public class DockerComposeV2Test {
```

Comment:
I am a bit unsure if we really need this specific V2 class, or if the other classes will generally deal with V2 going forward.

Reply:
due to we just set the compose v2 by default this test just makes sure the naming convention is taken into account